jp/components/general/update-guide.md

```diff
@@ -367,7 +367,7 @@ grid.getRowByIndex(0).expanded = false;
 ```
 *row.rowData* および *row.rowID* は非推奨であり、バージョン 13 で完全に削除されます。代わりに *row.data* と *row.key* を使用してください。
 
-* *onRowPinning* によって発行されたイベント引数の *row* プロパティ、および *onRowDragStart* によって発行されたイベント引数の *dragData* プロパティ、*onRowDragEnd* は [`RowType`]({environment:angularApiUrl}/interfaces/rowtype.html) を実装しています。 
+* *rowPinning* によって発行されたイベント引数の *row* プロパティ、および *onRowDragStart* によって発行されたイベント引数の *dragData* プロパティ、*onRowDragEnd* は [`RowType`]({environment:angularApiUrl}/interfaces/rowtype.html) を実装しています。 
```

Comment:
This is in a 11.1 to 12.0 upgrade guide. If you are sure the events were changed in 12.0, first this should be fixed in the EN and KR update guides. Also check if the `onRowDragEnd` & `onRowDragStart` in this same line should be changed.

Reply:
@ChronosSF,
It's fixed now